pkg/async/schedule/flytescheduler/interfaces/go_gf_wrapper.go

```diff
@@ -0,0 +1,19 @@
+package interfaces
+
+import (
+	"context"
+	"github.com/flyteorg/flyteadmin/pkg/repositories/models"
+	"time"
+)
+
+// GoGFWrapper Wrapper interface to the gogf framework which is used for scheduler
```

Comment:
this comment does not explain much, why do we need a wrappeR?

Reply:
For example, we should write that this wrapper handles scheduler common logic like jitter subtraction, time locking etc